File: app/src/main/java/ua/com/amicablesoft/android/wr/AddPowerlifterPresenter.java
package ua.com.amicablesoft.android.wr;

import android.content.Intent;

/**
 * Created by lapa on 03.10.16.
 */

public class AddPowerlifterPresenter {

    public boolean isLastNameValid(String lastName) {
        return lastName.length() > 1;
    }

    public boolean isNameValid(String name) {
        return name.length() > 1;
    }

    public void callWriteNewPowerlifter(String name, String lastName) {
        Repository repository = new Repository();
        repository.writeNewPowerlifter(name, lastName);

    }
}

File: app/src/main/java/ua/com/amicablesoft/android/wr/ui/AddPowerlifterPresenter.kt
/*
 * Created by Olha Tymoshenko on 5/25/18 3:43 PM.
 * Copyright (c) 2018. All rights reserved.
 */

package ua.com.amicablesoft.android.wr.ui

import ua.com.amicablesoft.android.wr.dal.Repository

class AddPowerlifterPresenter {

    fun isLastNameValid(lastName: String): Boolean {
        return lastName.length > 1
    }

    fun isNameValid(name: String): Boolean {
        return name.length > 1
    }

    fun callWriteNewPowerlifter(name: String, lastName: String) {
        val repository = Repository()
        repository.writeNewPowerlifter(name, lastName)

    }
}